test/bessel.jl

```diff
@@ -273,22 +273,29 @@ end
     end
 
     @testset "jinc function" begin
-        
+
         x1 = big"1.8"
         res1 = big"-0.116401402395234261196060221416100814277048989838340270527643894539781608507904800871249915340068849141209639297793338757544621191431028495364540423087719279432811811"
         x2 = big"0.00001"
         res2 = big"0.9999999998766299449914564074224855053704639002961349883799411022244488435839201448375732357069707105107902868318079100001712780899305533775708875568993566765287247356"
         x3 = big"0.02"
         res3 = big"0.9995066009475120760121935544369072906921634576001177613216005027724298234311657036029970452969291362640681823594159438475599412434802474217121139032145806264666152465837468052082455102325311416327533"
-        
+
         root1 = big"1.21966989126650445492653884746525517787935933077511212945638126557694328028076014425087191879391333148570410142424433731699"
         for T in [Float16, Float32, Float64]
             @test jinc(T(x1)) ≅ T(res1)
             @test jinc(T(0)) ≅ T(1)
-            x = root1 
+            x = root1
             # check the first root for negative and positive
-            @test jinc(T(x)) + T(1) ≅ T(1)
-            @test jinc(T(-x)) + T(1) ≅ T(1)
+            if T === Float16 && VERSION ≥ v"1.6.0-" && Base.BinaryPlatforms.arch(Base.BinaryPlatforms.HostPlatform()) == "armv7l"
```

Comment:
```suggestion
            if T === Float16 && VERSION ≥ v"1.6.0-" && Sys.ARCH === :armv7l
```
should work, right?

Reply:
I'm not sure that's always enough, `HostPlatform()` does some [normalization](https://github.com/JuliaLang/julia/blob/9f3265399227fbfc4f0160ec3592a9262bd3eb5f/base/cpuid.jl#L72-L89) of different possible architecture names, for example we have four different names for `armv7l`.